vnext/local-cli/generator-windows/templates/cpp/src/MainPage.idl

```diff
@@ -1,7 +1,7 @@
-namespace <%=ns%>
+namespace {{ namespace }}
```

Comment:
dot versus double-colon here too

Reply:
Since pre-existing I'll investigate and address if needed in separate pr.

Reply:
I assumed you were saying that it should be colon separated because today it is dot-separated... The namespace variable contains dot's. The issue is with the cpp file :) In my next PR I'll add proper naming and fix the CPP case.